packages/core/lib/services/util-crypto.js

```diff
@@ -26,7 +26,11 @@ export default {
       }
     },
     mnemonicValidate: (...params) => mnemonicValidate(...params),
-    mnemonicGenerate: (...params) => mnemonicGenerate(...params),
+    mnemonicGenerate: (...params) => {
+      console.log('request to generate mnemonic')
```

Comment:
if we're keeping this for debugging are there any interesting values in the params we should include in the log?

Reply:
I've removed this log